HTTPlayground/main.py

```diff
@@ -0,0 +1,39 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from HTTPlayground.views import not_found
+from HTTPlayground.urls import urls
+from HTTPlayground.settings import URL, PORT
+from HTTPlayground.base import FileReader, send_headers
+
+
+class SimpleRequestHandler(BaseHTTPRequestHandler):
+
+    def do_GET(self):
+        if self.path == '/favicon.ico':
+            return
```

Comment:
what does it lead to? Please check and fix if the behavior is incorrect.

Reply:
Micro optimization to not to allow further processing.
> what does it lead to? Please check and fix if the behavior is incorrect.

Reply:
optimization does make sense, but you need to handle it properly. E.g. by returning a 404 error.